tools/rpm2img

```diff
@@ -488,6 +692,19 @@ if [ "${OUTPUT_FMT}" == "vmdk" ] ; then
      -e "s/{{DATA_DISK_BYTES}}/${data_disk_bytes}/g" \
      > "${ova_dir}/${ovf}"
 
+  if [ "${UEFI_SECURE_BOOT}" == "yes" ] ; then
```

Comment:
It might be good to leave a comment as to why we are doing this in case we ever need to change things around ovf things.